runtime/src/snapshot_utils.rs

```diff
@@ -855,6 +839,77 @@ pub fn bank_from_snapshot_archives(
             None
         };
 
+    Ok((unarchived_full_snapshot, unarchived_incremental_snapshot))
+}
+
+pub fn bank_fields_from_snapshot_archives(
```

Comment:
Would you be able to add a few doc-comments (`///`) about this fn and _why_ someone would want to use it?